tests/nose_runner.py

```diff
@@ -132,3 +134,15 @@ def generate_tasks_input():
     while num_jobs:
         result = results.get()
         num_jobs -= 1
+
+    pytest_args = [
+        f"--local-dir={os.path.join(ytcfg.get('yt', 'test_data_dir'), 'answers')}",
+        "-c=pytest_answer.ini",
+        "--junitxml=unittests.xml",
+        "--answer-big-data",
+        # f"-n {int(os.environ.get('NUM_WORKERS', 6))}",
```

Comment:
Can you just enable it and see what happens? It will require adding `pytest-xdist` to `tests/test_requirements.txt` if you haven't done it already.

Reply:
Also please remove all the code related to nose from this script. When we talked about simultaneously running nose and pytest for some time, I meant running them as independent test suites, not in a single run.

Reply:
@Xarthisius sounds good! Should I instead move all of the pytest code to a file called, say, `pytest_runner.py` and leave the nose stuff as is? Then jenkins could run `python pytest_runner.py` or `python nose_runner.py` as needed? Further, would this entail commenting out the nose-specific test jobs in `.travis.yaml` (so that the two test suites are not run in a single run)?